packages/cli/src/workflows/workflows.services.ts

```diff
@@ -301,8 +301,8 @@ export class WorkflowsService {
 			);
 		}
 
-		if (isWorkflowHistoryLicensed()) {
-			await Container.get(WorkflowHistoryService).saveVersion(user, shared.workflow);
+		if (isWorkflowHistoryLicensed() && workflow.versionId !== shared.workflow.versionId) {
```

Comment:
@valya @krynble 
This change is breaking workflow activation/deactivation when `isWorkflowHistoryLicensed` returns true.